### ISSUE TITLE
Are the quoted accuracies correct in the README?

### ISSUE THREAD

**Issue by lewtun** [SanghunYun/UDA_pytorch#12]
Hello @SanghunYun,

This is a very clean implementation of UDA - thank you for sharing it!

I have a question about the accuracies quoted in the the README:

![Screen Shot 2021-01-19 at 4 09 13 pm](https://user-images.githubusercontent.com/26859204/105052919-ae2db480-5a70-11eb-85d0-658f94af56af.png)

From the screenshot, one sees the accuracy on the validation set is quite low (62.5%) while the value of 88.45% is the one quoted in the comparison against the published version of UDA. 

My question is why is there such a difference between the two values?

I ran your UDA script using the default values in the config

```bash
python main.py \  
      --cfg='config/uda.json' \
      --model_cfg='config/bert_base.json'
```

and I get significantly worse results:

* Eval Acc=0.500
* Accuracy : 0.661
* Max Accuracy : 0.881

Did you run your script for more steps or use a longer sequence length than that used in the defaults? More generally, can you share the hyperparameters you used to get the good results?

Thank you!

**Comment by prd-tin-nguyen** [comment]
You can check the setting of the paper's authors in their official code

**Comment by lewtun** [comment]
Thanks for the tip @prd-tin-nguyen! Closing this issue since my question was answered :)
